template/.github/workflows/android_deploy_production_to_playstore.yml

```diff
@@ -0,0 +1,50 @@
+name: Android - Deploy Production build to Play Store
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_deploy_android:
+    name: Build & Deploy Android
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
```suggestion
      - name: Checkout repository
        uses: actions/checkout@v3
```

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/163/commits/e681184449c6832f88b8ca3db6dc4ac489573a81